### ISSUE TITLE
ariaLabel defaults

### ISSUE THREAD

**Issue by bdc** [angular/material#557]
Since v0.5, I am getting a lot of warnings in the console about aria-label:
ARIA: Attribute "aria-label", required for accessibility, is missing on "md-button.md-raised.md-primary.md-default-theme" 
ARIA: Attribute "aria-label", required for accessibility, is missing on "md-checkbox.ng-pristine.ng-untouched.ng-valid" .

The docs at material.angularjs.org say this attribute is optional, and will use a button's or checkbox's text as a default.
